test/cmdlineTests/transient_storage_layout_value_types_interleaved_with_storage/input.json

```diff
@@ -0,0 +1,13 @@
+{
+	"language": "Solidity",
+	"sources": {
+		"fileA": {"urls": ["transient_storage_layout_value_types_interleaved_with_storage/in.sol"]}
+	},
+	"settings": {
+		"outputSelection": {
+			"fileA": {
+				"A": ["transientStorageLayout"]
```

Comment:
Request the `storageLayout` artifact here as well.

Reply:
Done.